### PR TITLE
chicv:0.1.0

### PR DESCRIPTION
**Pull request by skyzh**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: A minimal yet fully-customizable CV template.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

Note that this template did not add any new directives, and therefore the `lib.typ` is left blank and the template did not import the package. Obviously this does not align with the guideline in the readme of this repo. Do you have any guidelines for such templates? Thanks :)

### CODE REVIEW

File: packages/preview/chi-cv/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+license = "CC0-1.0"
+description = "A minimal and fully-customizable CV template."
+repository = "https://github.com/skyzh/chicv"
+keywords = ["cv"]
+categories = ["cv"]
+
+[template]
+path = "template"
+entrypoint = "cv.typ"
+thumbnail = "cv.png"
```
Comment (elegaanz) [inline packages/preview/chi-cv/0.1.0/typst.toml:15]: This won't work: the thumbnail path is relative to the root of the package, not to the template directory. Can you please move the `cv.png` file outside of this directory?

Comment (skyzh) [reply packages/preview/chi-cv/0.1.0/typst.toml:15]: sounds good, moved in https://github.com/typst/packages/pull/703/commits/c5cca8827558ad22cbe52f7dcb963248591a1e0d

**Comment by skyzh** [comment]
Ready for another round of review, thanks :) @elegaanz

**Comment by elegaanz** [comment]
Thanks :)

**Pull request merged**